### PR TITLE
Eye Changes for Ethnicities

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Gave **Draenei** `magic_no_pupils_eyes` and changed the brightness to 0.5.
- ~~Gave **Frostborn Dwarves** `magic_no_pupils_eyes`~~ and changed the brightness to 0.5.
  - Reverted back to `magic_eyes` in 997e635ab0489d202845889b9a3f9c706de1039a.
- Gave **Gilblin** and **Ice Trolls** `glowing_iris` and changed brightness to 1.
- Gave **Frost Vrykul**, **Kvaldir** and **Drust** `magic_no_pupils_eyes` and changed their brightness to 0.5.
- Readjusted elf eye brightness to 0.5, down from 1.
- Readjusted troll eye brightness to 0.5, up from 0.35.
- Readjusted **Frost Vrykul** and **Drust** eye color ranges so they include slightly lighter variations.

<details><summary>Images</summary>

image
image
image
image
image
image
image

</details>

### Implemented Suggestions
- Gave **Ice Trolls** more eye color variation in e96b155526bca4f9366f9a99cb0285b3fceb08e2 (suggested by @Remerod).
  - They may now generate with blue, cyan, purple, and pink eyes in addition to red and orange eyes. Red and orange eyes are, by far, the most common variation.
- Changed **Ice Troll** eyes to glowing_iris in 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).
- Changed **Gilblin** eyes to glowing_iris 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).

# How to test:
- View the changed ethnicities either using the Ruler Designer, [Debug Portrait Editor](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode), or by looking for their cultures in-game with the Character Finder (Draenei and Gilblin cannot be found using the last method).
- Provide feedback on the new eyes and if you think any other ethnicities should have their eyes changed or altered in some way (colors, brightness, eye type, etc.). You may use [this album](https://imgur.com/a/pQKUdkd) to make comparisons.

### CODE REVIEW

**Review by ValianBlue — APPROVED** [review]
_Overall the changes look great. The brightness adjustment to the elven eyes are my favorite in this change. They look more natural and fitting for their type._

_I'm curious about how the `magic_no_pupils_eyes` type would look with smaller pupils. This is not an issue, however._

File: common/ethnicities/wc_ethnicities_elven.txt [hunk 1]
```diff
@@ -370,7 +370,7 @@ high_elven = {
 	
 	#Eyes
 	special_eyes = {
-        1 = { name = magic_eyes      range = { 1 1 } }
+        1 = { name = magic_eyes      range = { 0.5 0.5 } }
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_elven.txt:373]: The glow of the DNA elves Kael'thas, Anasterian has not changed. Is it inteted?

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/ethnicities/wc_ethnicities_elven.txt (hunk 1 above)
Comment (ercarp) [reply common/ethnicities/wc_ethnicities_elven.txt:373]: Wasn't intended. Fixed in 617d9e53e1df8cb0acac34d0f0790dbcf08923f1.

**Review by Remerod — CHANGES_REQUESTED** [review]
So Overall i really like this change. but i do Have some comment's on the eye's

**Frostborn**
<details>
<summary>Click to expand</summary>

I think the Frostborn dwarf should have glowy eyes, but less so or maybe a change to their eye's, becouse currently they look a little souless to me.
image
image
Here is one on Falstad from HoTS with frostbornskin, i think his eye's look more alive
image
Generally Frostborn in WoW looks like they have a glow, but really a small one
image
I am honestly starting to think that the old eyes looked great
image

It is really hard to estimate what is a good eye for frostborn, since their is almost zero official art on them and the models are oldish. I do feel like their eye (as in eye white) is darker (blue) then their iris, who seem to be brighter and white-blueish. So perhaps something in between of the old and new eyes for them is better with darker eye and a lighter iris/pupil? 

</details>

On the topic of  **Frost Vrykul, Kvaldi**r and **Drust** , I think they look great! they look very eerie/spooky and it fit their theme very well! 

**Zandalari troll's** look great!

**Ice Troll**
<details>
<summary>Click to expand</summary>

I still is on the opinion that Ice Troll need's an pupil, not just an iris, as they currently have. Our version fit's the old wow models for Ice trolls, but the most modern Ice troll, King Malakk and newer artwork have them with not with just irises but with actual pupil's as well.
image
image
image
Put then again, there are some artwork with them having just irises and no pupil's
image
And then we have Warcraft 3 reforged stuff, where they seem to have only blue eye's. But still with both Irises and Pupils together. 
image
image
image
So it could be nice to add some eye colours perhaps? but mostly I'd like to see how they look with both pupil's and irises together, because they look nice as they are now but they stick out from the other non-zandalari trolls when they shouldn't, cause they are not more magical then any other troll kind.

</details>

**Gilblin**
<details>
<summary>Click to expand</summary>

Almost every Gilblin I have seen have both Irises and blackish pupil's in them, So i am unsure why ours don't.
image
image
image
image
image
image
image
Some of them do on the other hand have coloured eye-white. So i feel like they could use a change.

</details>

**Daenei Eredar** looks great and so does **Man'ari Eredar**

Provide feedback on the new eyes **and if you think any other ethnicities should have their eyes changed or altered in some way**

**Night Elf/Kaldorei**
<details>
<summary>Click to expand</summary>

So the Night Elf in game looks great at the moment! And i don't think they need to change... **But** in my opinion they **can** become better. Currently then look really close how they do in WoW. But we do know that how they look in WoW is not how Blizzard officialy depict them in their more detailed work. 
image
image
As stated, they look good, and close to the game version.
But i think most "great" art conception of Night Elf have more variation and nuance in their eyes.
image
It is hard to see, but there is an eye-white, but not shown in this picture but the eye-white do also often have a slight glow, there is the iris that is very bright, and this is the hard to see part, the pupil is outlined here, it is of course not black but kind of the same colour of the iris, other picture have the pupils more brightly glowing then the iris.
image
On this wc3 concept art you can clearly see a beautiful moonlight glow from the pupil, with a less but stil shining silvery moonish Iris and with a less but faintly glowing eye-white.
image
image
image
image
image
some female nelfs
image
image
While druid's clearly have different eyes, since they are supposed to be affected with druid magic. 
image
image
Malfurion's eyes
image
image
image
Tyrande's eyes.

And I don't think the NIght Elf's are the only only example elf's you could see a shiny pupil in their iris. Some High/Blood elf's actually is seen with dark pupil's in their art and wc3 stuff.
image
image
image
image
image
image
image
image
image
image
image
image
So there are plenty examples.

image
So I would like to suggest for a edited/possible new eye type for normal silver nelf eye's to be like in this picture that, 1. have a brighter light source as/(or over) a pupil, 2. have a less but still shining iris, that is not the same light as the pupil, but more silverish. 3 have the eye-white also have a slight glow but very feint. So all 3 should look from a distance like they meld together but if you look closely it all is different.

As i pointed out earlier, what we have now is good, it fit well with the wow/game look. And it is really close to what i am suggesting, but i feel if we could differentiate the different eye part's more for Night elves or elves in general it would look a little better in my opinion if we want them to look more realistic, or more in line of what Blizzard envisioned their eye's to be like.

But i stress out again, that this is just a extra suggestion.

</details>

**Comment by ercarp** [comment]
> I think the Frostborn dwarf should have glowy eyes, but less so or maybe a change to their eye's, becouse currently they look a little souless to me.

Well, you've clearly done your homework on this! I'll change them back to what they had before, you make some good points. The current ones may be better reserved for the **_Earthen_**:

<details><summary>Click to expand</summary>

image

</details>

> So it could be nice to add some eye colours perhaps? but mostly I'd like to see how they look with both pupil's and irises together, because they look nice as they are now but they stick out from the other non-zandalari trolls when they shouldn't, cause they are not more magical then any other troll kind.

I will add more eye color options for Ice Trolls, but I will make them rarer than the red eyes (which seem to be the most common, I'd say).

> I still is on the opinion that Ice Troll need's an pupil, not just an iris, as they currently have. Our version fit's the old wow models for Ice trolls, but the most modern Ice troll, King Malakk and newer artwork have them with not with just irises but with actual pupil's as well.

> Almost every Gilblin I have seen have both Irises and blackish pupil's in them, So i am unsure why ours don't.

I will ask @zumbak04 about adding this new type of eye. I think it would be good to have this as an option.

> So the Night Elf in game looks great at the moment! And i don't think they need to change... **But** in my opinion they **can** become better. Currently then look really close how they do in WoW. But we do know that how they look in WoW is not how Blizzard officialy depict them in their more detailed work.

What you're suggesting is nice, but it is very similar to magic_eyes. Therefore it may not be resource- or time-efficient to add a detail that would be barely visible in the character panel. You would only notice something like this when zooming in on the character's face.

**Comment by ercarp** [comment]
@Remerod **Gilblin** and **Ice Trolls** now use the new `glowing_iris` eyes.

<details><summary>Screenshots</summary>

image
image
image
image

</details>

**Review by Remerod — COMMENTED** [review]
I really like the new eye changes!!! They fit so well.

But i do have some remarks.
<details>
<summary>Click to expand</summary>

I think it looks really well with the new eye colours to!
image

But i dunno if the red eye's disappeared? or if they just look more orange now, some are clear orange but other's are more orange red.
image
image

But perhaps that fit's their looks?
image
So is it intended to be less red? because it was really red before, i don't really dislike the new lessened red.

Also King Malak have not gotten is new eye's, "sadge"
image

The Gilblin look's good with their new eye's
image
But dare i say where they are rich in quality they lack in quantity . I can only see the yellow-orange-ish & the blue eye colours in their randomnize appearance on the character design. and while it is not canon,  I would suggest to add this green-yellowish eye colour as a third normal appearance for Gilblin, found on google image search of Gilgoblin.
image

Overall i really like this change, good work!
</details>

File: common/dna_data/wc_elf_dna.txt [hunk 2]
```diff
@@ -106,7 +106,7 @@ lorthemar_theron_dna = { # by ercarp & TheDrPibb
 			eyelashes_accessory={ "normal_eyelashes" 181 "normal_eyelashes" 181 }
```
Comment (zumbak04) [inline common/dna_data/wc_elf_dna.txt:106]: You haven't changed the fel orc and undead eyes. Is it intended?
<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by ercarp** [comment]
> But i dunno if the red eye's disappeared? or if they just look more orange now, some are clear orange but other's are more orange red.
> So is it intended to be less red? because it was really red before, i don't really dislike the new lessened red.

It wasn't intended, it's just that colors interact slightly differently with the new eyes so they look a bit more toned down in terms of colors. But as you said, I don't think it's a bad thing necessarily.


> Also King Malak have not gotten is new eye's, "sadge"

Thanks for pointing that out, added in aec3c90b77bb3a5731ae05ba442101b2fd103790.

<details><summary>Screenshot</summary>

image

</details>

> But dare i say where they are rich in quality they lack in quantity . I can only see the yellow-orange-ish & the blue eye colours in their randomnize appearance on the character design. and while it is not canon, I would suggest to add this green-yellowish eye colour as a third normal appearance for Gilblin, found on google image search of Gilgoblin.

Added in aec3c90b77bb3a5731ae05ba442101b2fd103790. Gilblin may now generate with green/yellow eyes as well. I also bumped up the chance for them to generate with eyes other than red/orange.

<details><summary>Screenshots</summary>

image
image

</details>

File: common/dna_data/wc_elf_dna.txt (hunk 2 above)
Comment (ercarp) [reply common/dna_data/wc_elf_dna.txt:106]: Wasn't intended. Fixed in aec3c90b77bb3a5731ae05ba442101b2fd103790.

<details><summary>Click to expand</summary>

image
image

</details>

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Pull request merged**